src/frontend/src/lib/components/ui/MainLayout.svelte

```diff
@@ -0,0 +1,15 @@
+<main class="relative mx-auto max-w-screen-2.5xl pt-8 sm:flex sm:flex-row">
```

Comment:
Do you mind if we we name this component `SplitPane`?

I generally do so, so it would help me if I need to retrieve it, and also that way we use the same naming convention as in gix-cmp for example.

Reply:
Yes, for sure, will do!

P.S. I tried to use that GIX component but the requirements say that the central part is always "centered" for desktop, and I wasn't able to adapt it to the case

Reply:
done!